src/components/PairPage/PairForm/SPIModal/index.tsx

```diff
@@ -53,34 +64,43 @@ function SPIModal({ modalToggle, handleProviderChange, onClose, providerValue }:
         Simple Payments Integration
       </DialogTitle>
       <Typography variant="body2" className={classes.spiModalSubHeading}>
-        Select your Payment provider
+        Select Your Payment Provider
       </Typography>
       <DialogContent dividers>
         <FormControl component="fieldset">
           <RadioGroup
             aria-label="paymentProviderSelector"
+            id="paymentProviderSelector"
```

Comment:
Just so that these selectors remain stable and someone doesn't accidentally change them, can you please make it explicit that these are for tests using a convention like:
```suggestion
            data-test-id="paymentProviderSelector"
```

Reply:
Yep sure Matt, I will browse the codebase and try to use `data-test-id` instead of `id`.

Reply:
Fixed (I have updated all `id` to `data-test-id` for unit testing purpose)